Code/Framework/AzToolsFramework/Platform/Linux/AzToolsFramework/API/PythonLoader_Linux.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzToolsFramework/API/EditorPythonConsoleBus.h>
+#include <dlfcn.h>
+
+namespace AzToolsFramework::EmbeddedPython
+{
+    class PythonLoader
+    {
+    public:
+        PythonLoader()
+        {
+            m_embeddedLibPythonHandle = dlopen("libpython3.7m.so.1.0", RTLD_NOW | RTLD_GLOBAL);
+            if (!m_embeddedLibPythonHandle)
+            {
+                char* err = dlerror();
+                if (err)
+                {
+                    AZ_Error("PythonLoader", false, "Failed to load 'libpython' with error: %s\n", err);
+                }
+            }
+        }
+
+        ~PythonLoader() //override
+        {
+            if (m_embeddedLibPythonHandle)
+            {
+                dlclose(m_embeddedLibPythonHandle);
```

Comment:
I'm not sure if this is safe to do considering this can live in dynamic modules separately. If (someday) we unload a Gem with python bindings, but there are other gems that need it but are not unloaded, you can cause an issue. I think its fine not to do an unload for this scenario